lib/sensu-handler.rb

```diff
@@ -96,14 +96,17 @@ def api_request(method, path, &blk)
       if api_settings.nil?
         raise "api.json settings not found."
       end
-
-      req = net_http_req_class(method).new(path)
-      http = Net::HTTP.new(api_settings['host'], api_settings['port'])
+      domain = api_settings['host'].start_with?('http') ? api_settings['host'] : 'http://' + api_settings['host']
+      uri = URI("#{domain}:#{api_settings['port']}#{path}") 
+      req = net_http_req_class(method).new(uri)
```

Comment:
Line 99 is to make it backwards compatible.  If the host given is 'sensuapi.mywebsite.com' it will make it uri compatible.